src/Controls/tests/UITests/Tests/Issues/Issue20801.cs

```diff
@@ -0,0 +1,25 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.AppiumTests.Issues;
+
+public class Issue20801 : _IssuesUITest
+{
+	public override string Issue => "[Android] ImageButton Border does not fit to CornerRadius";
+
+	public Issue20801(TestDevice device)
+		: base(device)
+	{ }
+
+    [Test]
+	public void CornersShouldBeProperlyRounded()
+	{
+		this.IgnoreIfPlatforms(new TestDevice[] { TestDevice.iOS, TestDevice.Mac, TestDevice.Windows });
+
+		_ = App.WaitForElement("imageButton");
+
+		// The test passes if corners are properly rounded
+		VerifyScreenshot();
```

Comment:
Could we run this tests in all the platforms?. The corners are usually the most fragile property, if we have more tests validating it on all platforms, better.

Reply:
Sure, done!